maps_dashboards/public/components/add_layer_panel/add_layer_panel.tsx

```diff
@@ -40,23 +40,45 @@ export const AddLayerPanel = ({ setIsLayerConfigVisible, setSelectedLayerConfig
         opacity: 1,
         visibility: LAYER_VISIBILITY.VISIBLE,
       });
+    } else if (layerItem === DASHBOARDS_MAPS_LAYER_TYPE.DOCUMENT_LAYER) {
+      setSelectedLayerConfig({
+        iconType: 'document',
+        name: DASHBOARDS_MAPS_LAYER_TYPE.DOCUMENT_LAYER,
+        type: DASHBOARDS_MAPS_LAYER_TYPE.DOCUMENT_LAYER,
+        id: uuidv4(),
+        zoomRange: [0, 22],
+        opacity: 0.5,
+        visibility: LAYER_VISIBILITY.VISIBLE,
+        source: null,
+      });
     }
     setIsAddNewLayerModalVisible(false);
     setIsLayerConfigVisible(true);
   }
 
-  const availableLayers = Object.values(DASHBOARDS_MAPS_LAYER_TYPE).map((layerItem, index) => {
+  const dataLayers = [DASHBOARDS_MAPS_LAYER_TYPE.DOCUMENT_LAYER];
+  const dataLayerItems = dataLayers.map((layerItem) => {
+    return (
+      <EuiKeyPadMenuItem
```

Comment:
nit: Why does `referenceLayersItems` have `EuiFlexItem` but not `dataLayerItems`

Reply:
Updated!